.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  test-tarantool:
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8']
+
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Why do you use `master`? It may be unstable and the reason for the failure of this workflow.

Reply:
Because `master` is a current development branch.

Reply:
The `master` branch may be unstable. Why not `v2` that is the recommended version https://github.com/actions/checkout/tree/master#usage ?

Reply:
Sorry, missed your comment. I haven't found any difference for v1 and v2. Could you elaborate?
P.S. I'm not proficient in YAML programming, so your help would be appreciated.

Reply:
DDL uses `actions/checkout@master` for about a year without any problems, see https://github.com/tarantool/ddl/commit/4f5239a73edc7c310c0a5b7688355370b185f3ff#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R16

Reply:
Ok, but it doesn't mean that we will have no problems in the future ;) The `master` branch may get poor-tested changes with a high probability, of course, `v2` also may,  but it's still considered as a stable version. Anyway, I think it is not a big deal and we can fix it in some follow-up patches.

Reply:
It is general recommendation for using actions. In fact, after some time, when GitHub suggests `@v1`, `@v2` and so on, they start to suggest to stick on particular commit due to security reasons (a tag can be repushed). However that looks a bit weird, so I would prefer to stick on a tag, when security is not critical.